assets/stylesheets/gu-sass/colours.scss

```diff
@@ -76,7 +76,12 @@ $gu-colours: (
 
 	// External content palette
 	external-main-1:        #1c6326,
-	external-support-1:     #a9af2b
+	external-support-1:     #a9af2b,
+
+	// Support palette
```

Comment:
Is this the palette for the new design?

Reply:
Yes.

Reply:
👍

Reply:
Created a card for coming up with better names for the colours.